Schematic.podspec

```diff
@@ -46,4 +46,7 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Schematic/Classes/**/*'
 
+  s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

Comment:
@nihalahmed the main change is up here, where we added these on the pod spec level so you don't need to override on the pod file or your XCConfig